Clicker/Views/Cards/CardCell.swift

```diff
@@ -31,7 +31,7 @@ class CardCell: UICollectionViewCell {
     var timerLabel: UILabel!
     
     // MARK: - Data vars
-    var delegate: CardCellDelegate!
+    weak var delegate: CardCellDelegate!
```

Comment:
if these are `weak` should they be optional? when i refactored the delegates there was a lot of problems with making these optional tho bc of how often they're used (+ assumed to be there)